### PR TITLE
Migrate website to Jekyll generator

### PR DESCRIPTION
**Pull request by fpapon**
This is a first migration "as is" to Jekyll.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ Checkout:
...
 
-## Publish
+## Run
 
-To publish the local website to the production location (https://karaf.apache.org), you have to use:
+To start Jekyll local server, just run:
 
 ```
-mvn install scm-publish:publish-scm
+bundle exec jekyll serve
```
Comment (jbonofre) [inline README.md:37]: It's good to have the "server" mode, but generate mode and publish to scm should stay there.

**Review by jbonofre — CHANGES_REQUESTED** [review]
The conflict should be fixed. Thanks !

File: README.md (hunk 1 above)
Comment (fpapon) [reply README.md:37]: Ok, for deployment, I kept the pom.xml with the war packaging and the scm-publish.
I added more information in the README.

**Comment by fpapon** [comment]
@jbonofre conflicts are resolved, ready to review.

**Review by jbonofre — APPROVED** [review]

**Pull request merged**